### DISCUSSION TITLE
PORTAL ENGINE

### DISCUSSION THREAD

**Discussion by Yuki1602** [pimcore/pimcore#10833] [Q&A / Support]
when I multiselect the elements or download a collection it shows download generating but doesnt download anything, any single item when selected and downloaded works fine

**Comment by fashxp** [comment]
You need to make sure that your symfony messaging queue is processed...

**Reply by Yuki1602** [reply]
i ran the php bin/console messenger:consume -vv command and selected portal engine async still the same problem

**Reply by Yuki1602** [reply]
Any solution for this please

**Reply by fashxp** [reply]
does the download preparation finish and show you a download button in the notification? 
![image](https://user-images.githubusercontent.com/8792145/144247825-48d5e1c1-ab75-42b5-9aec-1d877e20a452.png)

do you see any thing in the output of the message consume?
